### PR TITLE
Fix voucher celery and errors

### PR DESCRIPTION
**Pull request by polesye**
Fixes for:
- https://liv-it.atlassian.net/browse/WEB-893
- https://liv-it.atlassian.net/browse/WEB-900
- https://liv-it.atlassian.net/browse/WEB-888

@avkoval @leha2000 please review.

### CODE REVIEW

**Comment by avkoval** [comment]
:+1:

File: lms/djangoapps/labster_vouchers/views.py [hunk 1]
```diff
@@ -60,17 +67,27 @@ def activate_voucher(request):
...
     if not form.is_valid():
-        messages.error(request, "Enter a valid voucher code.")
+        messages.error(request, _("Enter a valid voucher code."))
         return redirect(enter_voucher_url)
 
     code = form.cleaned_data['code']
 
     try:
         license_code = get_license(code)
+    except VoucherError as ex:
```
Comment (leha2000) [inline lms/djangoapps/labster_vouchers/views.py]: Is this Celery error handling?

**Comment by leha2000** [comment]
Where's the celery fix?

**Comment by polesye** [comment]
@leha2000 done.

**Comment by leha2000** [comment]
:+1:

**Pull request merged**